VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.ts

```diff
@@ -49,4 +50,8 @@ export class PrivateConsultationRoomControlsComponent extends HearingControlsBas
     canLeaveConsultation() {
         return this.participant?.status === ParticipantStatus.InConsultation;
     }
+
+    leave(confirmation: boolean) {
+        super.leave(confirmation, this.participantService.participants);
```

Comment:
Is participantService.participants relevant to consultations? I think it's just regarding the hearing.

Reply:
Not sure. I want to look closer at this leave functionality another time.